core-services/calendar/src/main/java/org/silverpeas/core/calendar/model/Priority.java

```diff
@@ -25,18 +25,16 @@
 package org.silverpeas.core.calendar.model;
 
 import java.io.Serializable;
+import java.util.HashMap;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.HashMap'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)